### PR TITLE
fixed menu buttons style.

### PR DESCRIPTION
**Pull request by Shivam-apna**
Please review and approve it.

### CODE REVIEW

File: src/components/Loader/loaderMenu.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="menu-bar" :class="{ focus: focus }">
```
Comment (ankitvats1) [inline src/components/Loader/loaderMenu.vue:2]: Remove focus.

File: src/components/Loader/loaderMenu.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    emits: ["toggle"],
+
+    computed: {
+      isElectron() {
+        return electronHelpers.isElectron()
+      },
+    },
+
+    created() {
+      const theme = settingsModule?.data.theme || "light"
```
Comment (ankitvats1) [inline src/components/Loader/loaderMenu.vue:39]: Remove this.

**Review by ankitvats1 — CHANGES_REQUESTED** [review]

**Comment by ankitvats1** [comment]
Update your branch.

**Comment by brsmxq** [comment]
@Shivam-apna  can you please review this?

**Pull request merged**